css/style.css

```diff
@@ -1,19 +1,140 @@
+.nav_bar {
+    background-color:whitesmoke;
+    position: fixed;
+    width: 100%;
+    border-bottom: 5px solid #DDD;
+}
+.nav_bar .nav_bar_ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+  }
+  
+  .nav_bar_ul li {
+      float: right;
+      align-content: center;
+      padding-inline-end: 20px;
+    }
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+  
+  .nav_bar_ul li a {
+    display: inline-block;
+    color: black;
+    text-align: center;
+    padding: 20px 10px;
+    text-decoration: none;
+  }
+  
+  .nav_bar_ul li a:hover:not(.active) {
+    background-color: green;
+  }
+  
+  .active {
+    font-style: italic;
+    background-color: rgb(146, 123, 123);
+  }
 
-body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
-}
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+  .logo-image{
+    display: block; 
+    height: 20px;
+    width: 50px;
+    margin-inline-start: 300px;}
+
+
+    .introduction{
+        display: block;
+        background-image: url(img/first-background.jpg);
+        background-position: center;
+        background-repeat: no-repeat;
+        background-size: cover;
+        height: 600px;
+        overflow:auto;
+    }
+    .intro_content{
+        display: block;
+        font-size: large;
+        color: rgb(240, 232, 232);
+        text-align: center;
+        box-sizing: border-box;
+        padding: 0 0;
+        margin: 0 auto;
+    }
+    .introduction h1{
+     margin-top: 20rem ;
+     letter-spacing: 5px;
+     font-weight: 2000px;
+     font-family: 'Bitter', serif;
+     text-align: center;
+    }
+    .introduction h3{
+        font-weight: 100;
+        letter-spacing: 5px;
+        font-family:'Bitter', serif ;
+    }
+    .button{
+        text-align: center;
+        margin: 50px;
+        font-size:x-large;
+        border-radius: 5px;  
+    }
+    .cases {
+        background-color: whitesmoke;
+        padding-bottom: 7px;
+        padding-top: 5px;
+    }
+    .cases_h1{
+        text-align: center;
+        font-size: x-large;
+        font-family:'Bitter', serif;
+        letter-spacing: 3px;
+        margin: 50px 100px;
+    }
+    .devices{
+        list-style-type: none;
+        display: flex;
+       justify-content: space-around;
+        
+    }
+    .social {
+     display: flex;
+     flex-direction: column;
+     justify-content: center;
+     align-content: center;
+
+
+    }
+    .social_media{
+        list-style-type: none;
+        display: flex;
+        
+    }
+    .social_media li img {
+        width: 50px;
+    }
+
+
+    @media (max-width: 328px){
```

Comment:
It'd be great if you could carry on working on the media queries so it scales nicely!